### PR TITLE
Set the "Scale window" larger and select the lock by default

### PR DESCRIPTION
**Pull request by coliss86**
I increased the size of the scale window in order to see better the values.
The "lock 🔒" is selected by default, and the button up and down increment by 1 unit instead of 0.1.

Before:
image

After:
image

### CODE REVIEW

File: src/main/com/marginallyclever/makelangelo/ScaleTurtlePanel.java [hunk 1]
```diff
@@ -31,16 +19,16 @@
 	/**
 	 * 
 	 */
+	@Serial
```
Comment (i-make-robots) [inline src/main/com/marginallyclever/makelangelo/ScaleTurtlePanel.java:22]: what is this for?

File: src/main/com/marginallyclever/makelangelo/ScaleTurtlePanel.java [hunk 2]
```diff
@@ -226,12 +217,9 @@ public static void main(String[] args) {
...
 
-		try {
-			JFrame frame = new JFrame(ScaleTurtlePanel.class.getSimpleName());
-			frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-			runAsDialog(frame,new Turtle());
-		} catch (Exception e) {
-			JOptionPane.showMessageDialog(null,e.getMessage(),"Error",JOptionPane.ERROR_MESSAGE);
-		}
+		JFrame frame = new JFrame(ScaleTurtlePanel.class.getSimpleName());
+		frame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
```
Comment (i-make-robots) [inline src/main/com/marginallyclever/makelangelo/ScaleTurtlePanel.java:221]: EXIT_ON_CLOSE causes system.exit() to be called, as discussed elsewhere.  DISPOSE_ON_CLOSE might be what you're looking for.

**Review by i-make-robots — COMMENTED** [review]
I wasn't trying to start a review.  but my comments stand.

File: src/main/com/marginallyclever/makelangelo/ScaleTurtlePanel.java (hunk 2 above)
Comment (coliss86) [reply src/main/com/marginallyclever/makelangelo/ScaleTurtlePanel.java:221]: the purpose of this modification is to remove the `catch` and let the exception be handled by the `main()` method which will display it in the console and exit in error if something wrong happen. Nothing related to `EXIT_ON_CLOSE` or else ;-)

**Pull request merged**